src/dev/build/tasks/os_packages/package_scripts/postinst_upgrade.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
```

Comment:
I believe the name of this file could be changed to `post_upgrade.sh` in order to follow the same logic of the others 😃

Reply:
Sure